README.md

```diff
@@ -21,32 +19,66 @@
 * They know how to connect the ViewController.swift file and understand it's view life cycle.
 * Can hook up IBOutlets and IBActions.
 
-## Outline / Notes
+## Simon Says: Do This Lab
+
+Remember the game [Simon](https://en.wikipedia.org/wiki/Simon_\(game\))? Simon was an electronic game popular in the 1980s. The toy would play a sequence of colors, and you had to press buttons corresponding to the colors in the same order as the sequence given.
+
+In this lab, you're going to recreate the game Simon.
+
+Don't worry: Most of the game logic has been written for you. You simply have to design the interface and set up the right Interface Builder outlets and actions to make the game work.
+
+Here's what the UI should look like when the game initially loads:
+
+![Simon UI](http://i.imgur.com/J70uWbC.png)
+
+There is one button in the center, **Start Game**, that will show a sequence of colors in a view just above the button. (Initially, this view is hidden from the user.) The UI also consists of four buttons, one in each corner, and each one a different color: Red, Green, Yellow, and Blue, as shown.
+
+There is also a hidden label beneath the **Start Game** button that will let the user know if they won or lost the game.
+
+When the user presses **Start Game**, a sequence of colors will be shown in the (initially hidden) view just above the **Start Game** button. After the sequence is shown, the user will be able to press the colored buttons. If they press them in the correct order, they win, and see this screen:
+
+![Simon: You won!](http://i.imgur.com/9ZXBrYH.png)
+
+If they press them in the wrong order, though, they will see this:
+
+![Simon: You lost!](http://i.imgur.com/xIryCfI.png)
+
+Let's get to it!
+
+Start by opening up `SimonSaysLab.xcworkspace`. You'll see that some parts of the view controller have already been started for you, as has a bit of the main UI (in `Main.storyboard`). You'll have to fill in the rest.
+
+## Game Engine
+
+A game engine has already been implemented for you. It is created as the `simonSaysGame` instance variable in the view controller. It has a number of methods already implemented for you that you will find useful:
+
+* There are `guessRed()`, `guessGreen()`, `guessYellow()`, and `guessBlue()` methods written. You can call these when the user makes a guess.
+* There is a `gameWon()` method that will return `true` if the user has made the correct guesses, and `false` if they haven't.
+
```

Comment:
I like the idea of including a screenshot here, showing them how they can call on these various methods.

![guess](http://i.imgur.com/ZJEqpnR.png?1)